src/newlines.cpp

```diff
@@ -2223,6 +2249,41 @@ static void newlines_brace_pair(chunk_t *br_open)
             }
             chunk_flags_set(br_open, PCF_ONE_LINER);         // set the one liner flag if needed
             chunk_flags_set(br_close, PCF_ONE_LINER);
+
+            if (  options::code_width() > 0
+               && br_close->column > options::code_width())
+            {
+               // is too long
+               // it is not possible to make an one_liner
+               // because the line would be too long
+               chunk_flags_set(br_open, PCF_NOT_POSSIBLE);
+               // restore the code
+               size_t  count;
+               chunk_t tmp;
+               chunk_t *current = br_open;
+
+               for (count = 0; count < saved_chunk.size(); count++)
+               {
+                  tmp = saved_chunk.at(count);
+
+                  if (tmp.orig_line != current->orig_line)
```

Comment:
Is it possible that `current` is a nullptr at this place?

Reply:
NO.
The array was fill just before.